Assets/Mirror/Components/Experimental/NetworkTransformBase.cs

```diff
@@ -0,0 +1,442 @@
+        [SyncVar]
+        public bool syncRotation = true;
+
+        [Tooltip("Set to true if scale should be synchronized")]
+        [SyncVar]
+        public bool syncScale = true;
+
+        [Header("Interpolation")]
+
+        [Tooltip("Set to true if position should be interpolated")]
+        [SyncVar]
+        public bool interpolatePosition = true;
+
+        [Tooltip("Set to true if rotation should be interpolated")]
+        [SyncVar]
+        public bool interpolateRotation = true;
+
+        [Tooltip("Set to true if scale should be interpolated")]
+        [SyncVar]
+        public bool interpolateScale = true;
+
+        // Sensitivity is added for VR where human players tend to have micro movements so this can quiet down
+        // the network traffic.  Additionally, rigidbody drift should send less traffic, e.g very slow sliding / rolling.
+        [Header("Sensitivity")]
+
+        [Tooltip("Changes to the transform must exceed these values to be transmitted on the network.")]
+        [SyncVar]
+        public float localPositionSensitivity = .01f;
+
+        [Tooltip("If rotation exceeds this angle, it will be transmitted on the network")]
+        [SyncVar]
+        public float localRotationSensitivity = .01f;
+
+        [Tooltip("Changes to the transform must exceed these values to be transmitted on the network.")]
+        [SyncVar]
+        public float localScaleSensitivity = .01f;
+
+        [Header("Diagnostics")]
+
+        // server
+        public Vector3 lastPosition;
+        public Quaternion lastRotation;
+        public Vector3 lastScale;
+
+        // client
+        // use local position/rotation for VR support
+        [Serializable]
+        public struct DataPoint
+        {
+            public float timeStamp;
+            public Vector3 localPosition;
+            public Quaternion localRotation;
+            public Vector3 localScale;
+            public float movementSpeed;
+
+            public bool isValid => timeStamp != 0;
+        }
+
+        // Is this a client with authority over this transform?
+        // This component could be on the player object or any object that has been assigned authority to this client.
+        bool IsOwnerWithClientAuthority => hasAuthority && clientAuthority;
+
+        // interpolation start and goal
+        public DataPoint start = new DataPoint();
+        public DataPoint goal = new DataPoint();
+
+        // local authority send time
+        public float lastClientSendTime;
+        public float lastServerSendTime;
+
+        void FixedUpdate()
+        {
+            // if server then always sync to others.
+            if (isServer)
+            {
+                // let the clients know that this has moved
+                if (HasEitherMovedRotatedScaled())
+                {
+                    RpcMove(targetTransform.localPosition, targetTransform.localRotation, targetTransform.localScale);
+                    lastServerSendTime = Time.time;
```

Comment:
`lastServerSendTime` isnt used anywhere, can we remove it?

Reply:
Seems like `lastClientSendTime` isnt used either. Should they be checked against `syncInterval` somewhere?

Reply:
Done